package.json

```diff
@@ -47,7 +47,44 @@
     "extends": "stylelint-config-recommended-scss",
     "plugins": [
       "stylelint-scss"
-    ]
+    ],
+    "rules": {
+      "declaration-property-unit-blacklist": [
```

Comment:
The upcoming [Stylelint v13.7.0 release](https://github.com/stylelint/stylelint/issues/4854) will [`Replace *-whitelist/*-blacklist/*-requirelist rules with *-allowed-list/*-disallowed-list/*-required-list rules`](https://github.com/stylelint/stylelint/pull/4845) as to not perpetuate white supremacy and racism. 

I think we should wait to merge this until the release is out and we can update to the new version and rule names. It looks like they're aiming for later this week.

Reply:
This release appears to have been delayed. I've opened https://github.com/IBM/carbon-addons-iot-react/issues/1393 to track and complete this work once the release is published. I'm okay with merging this as-is for now as these issues are blocked by the stylelint package.